sdk-base/src/main/java/com/mapbox/maps/plugin/ConfigProperties.kt

```diff
@@ -4,142 +4,6 @@ package com.mapbox.maps.plugin
 
 import android.graphics.drawable.Drawable
 
-/**
- * The map rendering mode. By default, it is set to Continuous so the map will render as data arrives from the network and react immediately to state changes.
- *
- * @param value
- */
-enum class MapMode(val value: String) {
-  /**
-   * Continuously updating map.
-   */
-  CONTINUOUS("continuous"),
-  /**
-   * A once-off still image of an arbitrary viewport.
-   */
-  STATIC("static"),
-  /**
-   * A once-off still image of a single tile.
-   */
-  TILE("tile"),
-}
-
-/**
- * The map context mode. This can be used to optimizations if we know that the drawing context is not shared with other code.
- *
- * @param value
- */
-enum class ContextMode(val value: String) {
```

Comment:
I think this should not be removed (and all below).
Just `MapMode` should be removed.

Reply:
These were generated from map and resource configurations, but these classes are manually written now, so these classes are outdated and unused.